pages/home.js

```diff
@@ -191,70 +191,79 @@ export default function Home(props) {
     }
   }
   return (
-    <div
-      data-testid="homeContent"
-      id="homeContent"
-      className="container grid grid-cols-1 gap-y-5 mx-auto sm:flex sm:justify-center sm:gap-x-5"
-    >
-      <Container style="text-center p-4 flex flex-col drop-shadow md:w-96">
-        <h2 className="text-opacity-75 text-black font-bold text-2xl">
-          {t.createRoomTitle}
-        </h2>
-        <h3 className="text-opacity-75 text-black text-xl">
-          {t.createRoomDesc}
-        </h3>
-        <form
-          data-testid="createRoomForm"
-          onSubmit={onCreateHandler}
-          className="flex flex-col justify-between h-full items-center"
-        >
-          {createRoomError ? (
-            <ErrorLabel message={createRoomError}></ErrorLabel>
-          ) : undefined}
-          <TextInput
-            id="owner"
-            label={t.createRoomLabel}
-            placeholder={t.createRoomPlaceholder}
-          />
-          <button
-            type="submit"
-            className="w-max font-display text-white bg-[#318000] hover:bg-[#1D4D00] active:bg-[#102900] py-3 px-5 rounded mt-12 focus:drop-shadow focus:ring-2 focus:ring-gray-600 border border-[#458259] text-[22px] leading-8 [text-shadow:1px_2px_0px_#333]"
+    <>
+      {router.query.error ? (
```

Comment:
This is a bit quirky and a security risk.
Perhaps try returning error=true or an error code in the query string.

Then set the error message to something generic